spec/features/courses/details_spec.rb

```diff
@@ -215,4 +217,65 @@
     .to be_displayed(provider_code: 'ZZ', course_code: 'ZZZ')
     expect(course_details_page.title.text).to eq 'Page not found'
   end
+
+  describe 'allocations' do
+    let(:course) do
+      build :course,
+            provider: provider,
+            recruitment_cycle: next_recruitment_cycle
+    end
+
+    context 'when the course is in the next recruitment cycle' do
+      scenario 'displays no restrictions' do
+        course_details_page.load_with_course(course)
+        expect(course_details_page.allocations_info).to have_content(
+          'Recruitment is not restricted'
+        )
+      end
+
+      context 'when the course is Physical Education' do
+        let(:course) do
+          build :course,
+                provider: provider,
+                recruitment_cycle: next_recruitment_cycle,
+                subjects: ["Secondary", "Physical education"]
+        end
+
+        scenario 'displays no restrictions' do
+          course_details_page.load_with_course(course)
+          expect(course_details_page.allocations_info).to have_content(
```

Comment:
do you not need to check there is a link?

Reply:
Felt like overkill